components/StyledDELButton.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+export const DelButton = styled.div`
```

Comment:
Wouldn't hurt to just call this component `DeleteButton` ;)

Reply:
:) will change it in a second :)